### PR TITLE
Adding unittests for several csv import related timestamp / datetime edge cases

### PR DESCRIPTION
**Pull request by jaegeral**
The PR adds several test data / csv files that have numerous edge cases for timestamp / datetime variations.

The unittests do not fail anything, but they are useful moving forward if someone wants to start tackling the problem, the expected outputs can be adjusted.

### CODE REVIEW

**Comment by jaegeral** [comment]
This PR is in relation with https://github.com/google/timesketch/issues/3173

File: timesketch/lib/utils_test.py [hunk 1]
```diff
@@ -232,6 +232,86 @@ def test_timestamp_is_ISOformat(self):
...
+                self.assertIsNotNone(item["timestamp"])
+                self.assertEqual(item["timestamp"], 1406253661000000)
+                self.assertIsNotNone(item["datetime"])
+                self.assertEqual(item["datetime"], "2014-07-25T02:01:01+00:00")
+            elif item["data_type"] == "Whitespace datetime":
+                self.assertIsNotNone(item["timestamp"])
+                self.assertEqual(item["datetime"], "2016-07-25T02:01:01+00:00")
+                self.assertIsNotNone(
+                    item["datetime"]
+                )  # TODO: This should not be a space
```
Comment (jkppr) [inline timesketch/lib/utils_test.py:263]: Is this a TODO still relevant? Can you link it to an issue?

**Review by jkppr — APPROVED** [review]
lgtm

File: timesketch/lib/utils_test.py (hunk 1 above)
Comment (jaegeral) [reply timesketch/lib/utils_test.py]: removed

**Comment by jaegeral** [comment]
Added some more tests, please re-review @jkppr thx

**Review by jkppr — APPROVED** [review]
lgtm
